src/libraries/System.Private.Xml/src/System/Xml/Schema/Inference/Infer.cs

```diff
@@ -113,7 +113,6 @@ public sealed class XmlSchemaInference
         private readonly XmlNamespaceManager _namespaceManager;
         //private Hashtable schemas;    //contains collection of schemas before they get added to the XmlSchemaSet xsc
         //private bool bRefine = false; //indicates if we are going to infer or refine schema when InferSchema is called
```

Comment:
Delete these commented out fields also?

Reply:
I wanted to leave out any cosmetic or unrelated changes and only target lines of code that matter for the fixes. For other code clean up/potential formatting like this, feel free to fix up separately if you like, but I would prefer to leave that out of this PR.